content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -96,27 +97,31 @@ The command has the following parameters:
     set for the node with ID 1, with a new rack ID. Otherwise, the node
     will get the same rack ID as the original node.
 
+Example:
+```src
+$ rladmin cluster recover filename /tmp/persist/ccs/ccs-redis.rdb node_uid 1
+```
 Once the command execution is complete, this node will replace the node
 with ID 1 from the old cluster.
 
-You can run the rladmin status command to verify that the cluster was
+You can run the `rladmin status` command to verify that the cluster was
 created based on the old cluster configuration, and all other nodes and
 databases are pending recovery.
 
 ### Recovering the other nodes
 
 For each of the other servers that are going to act as RS nodes, run the
-cluster join command from the rladmin CLI prompt in order to join the
+`cluster join` command from the rladmin CLI prompt in order to join the
 node to the recovered cluster, and specify which of the nodes in the old
 cluster this node is going to replace. The command has the following
 parameters:
 
+1. nodes - mandatory. The IP address of a node in the cluster this
+    node is joining. Alternatively, the name parameter can be used.
```

Comment:
@Paz-Yanover  Who needs to review this addition?

Reply:
@ushachar - who you want to review and approve it from your side?